src/api/app/assets/stylesheets/webui2/sticky-footer.scss

```diff
@@ -8,4 +8,13 @@ body {
 
 #footer {
   background-color: $gray-200;
+
+  & a {
+    font-size: .9rem;
+    color: $main_green;
+
+    &:hover {
+        color: $main_blue;
+    }
+}
```

Comment:
There are [color classes](https://getbootstrap.com/docs/4.1/utilities/colors/#color) for this.

Reply:
Those doesn't work with a hovering link, do they?

Or you mean adding a .text-* class to each link in the footer?

Reply:
Not to every link but to the container for instance.

Reply:
Doesn't work, the text would be green for every single thing there...